config/configunmarshaler/defaultunmarshaler.go

```diff
@@ -168,9 +168,14 @@ func unmarshalService(srvRaw map[string]interface{}) (config.Service, error) {
 	srv := config.Service{
 		Telemetry: config.ServiceTelemetry{
 			Logs: config.ServiceTelemetryLogs{
-				Level:       zapcore.InfoLevel,
-				Development: false,
-				Encoding:    "console",
+				Level:             zapcore.InfoLevel,
```

Comment:
Make sure these are similar with https://github.com/uber-go/zap/blob/master/config.go#L120 except Encoding which was "console" before and we don't want to break things in the PR.

Reply:
Thanks for pointing it out, add a test to protect it, see TestDefaultLoggerConfig